index.html

```diff
@@ -0,0 +1,853 @@
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/lowergrid2.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/lowergrid3.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/lowergrid4.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/lowergrid5.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/lowergrid6.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/lowergrid7.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="top-ftr flex">
+        <div class="ftr-icons flex">
+          <i class="fa fa-facebook-official" aria-hidden="true"></i>
+          <i class="fa fa-twitter" aria-hidden="true"></i>
+          <i class="fa fa-youtube-play" aria-hidden="true"></i>
+          <i class="fa fa-instagram" aria-hidden="true"></i>
+          <i class="fa fa-pinterest-square" aria-hidden="true"></i>
+          <i class="fa fa-rss" aria-hidden="true"></i>
+          <i class="fa fa-envelope" aria-hidden="true"></i>
+        </div>
+        <div class="ftr-links flex">
+          <a href="#">Events</a>
+          <a href="#">About</a>
+          <a href="#">Advertise</a>
+          <a href="#">Jobs</a>
+          <a href="#">contacts</a>
```

Comment:
- [ ] **OPTIONAL** To make it look consistent, you can capitalize "Contacts", or make all of these elements uppercase as in the original